psycopg_c/psycopg_c/types/uuid.pyx

```diff
@@ -0,0 +1,138 @@
+cimport cython
+
+from types import ModuleType
+from cpython.bytes cimport PyBytes_AsString
+from cpython.long cimport PyLong_FromUnsignedLongLong
+
+cdef extern from "Python.h":
+    # PyUnicode_AsUTF8 was added to cpython.unicode in 3.1.x but we still
+    # support 3.0.x
```

Comment:
Isn't this comment out of date? We only support 3.8+ and we never supported anything before 3.6.

Reply:
That's about the Cython version, not the Python version. `{impl: c, python: "3.9", ext: min, postgres: "postgres:15"}` uses it, see https://github.com/psycopg/psycopg/blob/master/tests/constraints.txt#L34.

I probably need to mention it in the comment if it isn't clear.

Reply:
Sorry, it was me to misread that. We can bump to a later version of Cython, being on 3.0 it's not a major requirement. But we can leave it out of this MR.